### DISCUSSION TITLE
#100DaysOfGatsby Challenge 2 - Did I miss the info?

### DISCUSSION THREAD

**Discussion by GrfxGuru** [gatsbyjs/gatsby#29296] [Help]
Did I miss it being posted or are we waiting for it to appear?
Thanks.

**Comment by Aereli** [comment]
Here ya go, it showed up on Jan 21st. 
https://www.gatsbyjs.com/blog/tags/100-days-of-gatsby/

**Reply by GrfxGuru** [reply]
Thank you so much!

**Reply by Kevpedia** [reply]
yeah, doesn't look like the email list works @hashimwarren ... 2nd time I never got any email, had to go manually check the blog
